samples/jvm/src/main/java/guru/stefma/cleancomponents/usecase/sample/jvm/Sample.kt

```diff
@@ -43,44 +46,45 @@ class GetUserUseCase(
  */
 @UseCase
 class GetUsePointsForUserIdUseCase(
-        private val getUser: GetUser,
-        override val executionScheduler: Scheduler,
-        override val postExecutionScheduler: Scheduler
+    private val getUser: GetUser,
+    override val executionScheduler: Scheduler,
+    override val postExecutionScheduler: Scheduler
 ) : SingleUseCase<Int, GetUsePointsForUserIdUseCase.Params> {
 
     override fun buildUseCase(params: GetUsePointsForUserIdUseCase.Params): Single<Int> {
         return getUser.buildUseCase(
-                guru.stefma.cleancomponents.usecase.sample.jvm.GetUserUseCase.Params(params.userId))
-                .flatMap {
-                    Single.just(it.authorizationKey)
-                            .map { 99 }
-                }
+            GetUserUseCase.Params(params.userId)
+        ).flatMap {
+            Single.just(it.authorizationKey)
+                .map { 99 }
+        }
     }
 
     data class Params(val userId: String)
 }
 
 @UseCase
-class GetUserCoroutineUseCase : CoroutineUseCase<User, GetUserCoroutineUseCase.Params> {
+class GetUserCoroutineUseCase(
+    override val callbackDispatcher: CoroutineContext = Dispatchers.IO
```

Comment:
Please use a different Dispatcher.
The `IO` one should only be used for IO operations, right?

Reply:
I guess getting the user would normally get user from database or network, which is IO, no?

Reply:
wrong, callback is no IO operation